src/test/kotlin/no/nav/syfo/api/endpoints/ArbeidsuforhetEndpointsSpek.kt

```diff
@@ -91,16 +93,25 @@ object ArbeidsuforhetEndpointsSpek : Spek({
             )
             val personIdent = ARBEIDSTAKER_PERSONIDENT.value
 
-            suspend fun createForhandsvarsel() = vurderingService.createVurdering(
+            suspend fun createVurdering(type: VurderingType, document: List<DocumentComponent>) = vurderingService.createVurdering(
                 personident = PersonIdent(personIdent),
                 veilederident = VEILEDER_IDENT,
-                type = VurderingType.FORHANDSVARSEL,
+                type = type,
                 begrunnelse = begrunnelse,
-                document = forhandsvarselDocument,
+                document = document,
                 gjelderFom = null,
                 callId = UUID.randomUUID().toString(),
             )
 
+            suspend fun createExpiredForhandsvarsel() {
+                Varsel.svarfristDager = 0
+                createVurdering(
+                    type = VurderingType.FORHANDSVARSEL,
+                    document = forhandsvarselDocument
+                )
+                Varsel.svarfristDager = 21
+            }
```

Comment:
Det virker litt som anti pattern å endre klassen før man lager et objekt og så endrer klassen tilbake, i stedet for å bare endre objektet 🤔

Reply:
Ja, fant en bedre måte å gjøre det på nå. Oppretter i stedet et utløpt forhåndsvarsel direkte med repository før test av POST avslag.